### PR TITLE
Vdo.ai

### PR DESCRIPTION
**Pull request by cpuBird**
VDO.AI : Implemented VDO.AI in amp-ad 
✨ New feature

### CODE REVIEW

**Comment by zhouyx** [comment]
to @ampproject/wg-ads

File: ads/vdo.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import {validateData, writeScript} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function vdo(global, data) {
+  /*eslint "google-camelcase/google-camelcase": 0*/
+  console.log(global, data);
```
Comment (powerivq) [inline ads/vdo.js]: Do you need this for production?

Comment (cpuBird) [reply ads/vdo.js]: Thanks for identifying this piece of code. We would like to remove this for production purposes. This was being used by us during development stage. Can you please remove this from your end? That would be great :D

Comment (cpuBird) [reply ads/vdo.js]: I have updated the code myself. No need for further modifications. Please have a final look. Thanks !

**Comment by powerivq** [comment]
@cpuBird Hi. I approved the PR, but
- CLA needs to be resolved. See comments above.
- Lint is failing. Please fix the issue.

**Comment by cpuBird** [comment]
@googlebot I signed it!

**Comment by cpuBird** [comment]
Hi @powerivq,
Thanks for approving the PR. 
1. I've signed the CLA.
2. The linting issue mentioned above was because of the CRLF line endings in windows. I've changed that to LF and the issue was resolved. I've commited the code pls have a look and tell me if anything required. :)

**Comment by powerivq** [comment]
> Hi @powerivq,
> Thanks for approving the PR.
> 
> 1. I've signed the CLA.
> 2. The linting issue mentioned above was because of the CRLF line endings in windows. I've changed that to LF and the issue was resolved. I've commited the code pls have a look and tell me if anything required. :)

Not sure why but the bot did not pick up your CLA. Can you respond "I signed it" again? Also, the build issue is still there.

**Comment by cpuBird** [comment]
> > Hi @powerivq,
> > Thanks for approving the PR.
> > 
> > 1. I've signed the CLA.
> > 2. The linting issue mentioned above was because of the CRLF line endings in windows. I've changed that to LF and the issue was resolved. I've commited the code pls have a look and tell me if anything required. :)
> 
> Not sure why but the bot did not pick up your CLA. Can you respond "I signed it" again? Also, the build issue is still there.

Hi @powerivq ,
CLA is accepted and I've also fixed the linting issues

**Comment by powerivq** [comment]
@cpuBird plz ping me when it passes. I see it still failing.

**Comment by cpuBird** [comment]
Hi @powerivq , 
Sorry for delay in revert. You told me that I have linting issues in commets above so I fixed them using the following commands mentioned on - https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md : 
`gulp lint --local_changes --fix` &
`gulp prettify --local_changes --fix`

I would appreciate if you can help me figure the exact issue so that i can fix it.

**Comment by powerivq** [comment]
@cpuBird You can click on the travis build result https://travis-ci.org/ampproject/amphtml/jobs/629961620?utm_medium=notification&utm_source=github_status

```
Type checking failed:
ads/vdo.js:26: ERROR - [JSC_POSSIBLE_INEXISTENT_PROPERTY] Property unitid never defined on Object
    unitData: data.unitid,
                   ^^^^^^
ads/vdo.js:35: ERROR - [JSC_POSSIBLE_INEXISTENT_PROPERTY] Property tagname never defined on Object
    ADFRAME_URL: 'https://a.vdo.ai/core/' + data.tagname + '/adframe.js' };
                                                 ^^^^^^^
```

**Comment by cpuBird** [comment]
Hi @powerivq ,
Please have a look. I've fixed the issues with failing build.

**Comment by powerivq** [comment]
@cpuBird You can look at the build status badge below. Everything except `owners-check` should succeed (showing a tick rather than X). You can check the `Details` button on `continuous-integration/travis-ci/pr — The Travis CI build failed` to see what went wrong. The local and remote test failures are related to your changes. The saucelabs seems to be a flaky test that would succeed.

**Comment by cpuBird** [comment]
@powerivq Done

**Comment by honeybadgerdontcare** [comment]
Drive by comment: if you're adding new files please have the copyright be the current year. In this instance it may be the first `2020` copyright: yeah!

**Comment by cpuBird** [comment]
Both done.

**Pull request merged**